frontend/components/vault/Vault.vue

```diff
@@ -157,16 +157,36 @@
                 </div>
             </Alert>
             <TextBlock class="mt-4">
-                This vault was liquidated <TimeTill :date="vaultTransaction.pastLiquidations[0].liquidationDate" /> in
-                the transaction <FormatAddress :value="vaultTransaction.pastLiquidations[0].transactionHash" />.
+                This vault was liquidated in the following transactions:
+                <table class="table-auto mt-4">
```

Comment:
How will this table look in dark mode? Have a look at #311 where I added a similar table and added styling for light and dark mode!